lib/Alchemy/Phrasea/Authentication/Provider/Github.php

```diff
@@ -144,7 +144,7 @@ public function onCallback(Request $request)
 
         $data = @json_decode($response->getBody(true), true);
 
-        if (JSON_ERROR_NONE !== json_last_error()) {
+        if (JSON_ERROR_NONE !== json_last_error() && $data != null) {
```

Comment:
what does it mean ? The opposite test (no pb) is (last_err==none || data==null), which means "there can be an error, but if the result is null it's ok" ?
Does it fix a known bug or a failing test ?
Is it related with php7 ?

btw : writing "yoda code" for "not equal" is just useless and make code unreadable. Please remove ALL "yoda" when you see some, as we will strictly prohibit this stupidity for new code !